pytest/tests/stress/perf_ft_transfer.py

```diff
@@ -0,0 +1,471 @@
+        if self.transaction_id is None or self.ttl <= 0:
+            if self.transaction_id is not None:
+                logger.warning(f"transaction {self.transaction_id} expired, submitting a new one!")
+            (self.transaction_id, self.caller) = self.send(node, block_hash)
+            self.expiration = time.time() + DEFAULT_TRANSACTION_TTL_SECONDS
+            self.ttl = DEFAULT_TRANSACTION_TTL_SECONDS
+            return False # almost guaranteed to not produce any results right now.
+        caller = ACCOUNTS[self.caller]
+        logger.debug(f"checking {self.transaction_id} from {caller.key.account_id}")
+        tx_result = node.json_rpc('tx', [self.transaction_id, caller.key.account_id])
+        if self.is_success(tx_result):
+            self.outcome = tx_result
+            return True
+        return False
+
+    def send(self, block_hash):
+        return (self.transaction_id, self.caller)
+
+    def is_complete(self):
+        return self.outcome is not None
+
+    def is_success(self, tx_result):
+        success = 'error' not in tx_result
+        if not success:
+            logger.debug(f"transaction {self.transaction_id} for {self.caller} is not successful: {tx_result}")
+        # only set TTL if we managed to check for success or failure...
+        self.ttl = self.expiration - time.time()
+        return success
+
+
+class DeployFT(Transaction):
+    def __init__(self, account, contract):
+        super().__init__()
+        self.account = account
+        self.contract = contract
+
+    def send(self, node, block_hash):
+        account = ACCOUNTS[self.account]
+        logger.warning(f"deploying FT to {account.key.account_id}")
+        wasm_binary = utils.load_binary_file(self.contract)
+        tx = transaction.sign_deploy_contract_tx(
+            account.key,
+            wasm_binary,
+            account.use_nonce(),
+            block_hash
+        )
+        result = node.send_tx(tx)
+        return (result["result"], self.account)
+
+
+class TransferFT(Transaction):
+    def __init__(self, ft, sender, recipient, how_much = 1):
+        super().__init__()
+        self.ft = ft
+        self.sender = sender
+        self.recipient = recipient
+        self.how_much = how_much
+
+    def send(self, node, block_hash):
+        (ft, sender, recipient) = ACCOUNTS[self.ft], ACCOUNTS[self.sender], ACCOUNTS[self.recipient]
+        logger.debug(f"sending {self.how_much} FT from {sender.key.account_id} to {recipient.key.account_id}")
+        args = {
+            "receiver_id": recipient.key.account_id,
+            "amount": str(int(self.how_much)),
+        }
+        tx = transaction.sign_function_call_tx(
+            sender.key,
+            ft.key.account_id,
+            "ft_transfer",
+            json.dumps(args).encode('utf-8'),
+            # About enough gas per call to fit N such transactions into an average block.
+            int(GAS_PER_BLOCK // TRANSACTIONS_PER_BLOCK),
```

Comment:
I don't think the amount of attached gas has any effect on the number of transactions that will be included in the block as the block fullness is computed based on the burnt gas.

I think we should just use a constant number here instead that is enough to cover the cost of that FT transfer, something like 50TGas should be more than enough.

We can then remove `TRANSACTIONS_PER_BLOCK` constant as it is not used anywhere and is a bit misleading.

Reply:
Fair, I think it might make sense to make this into an CLI argument while I’m changing this.